src/drivers/QtTestDriver.cpp

```diff
@@ -4,28 +4,81 @@
 #include <QTextStream>
 #include <QTemporaryFile>
 
+#include <iostream>
+
 namespace cucumber {
 namespace internal {
 
+// wraps the QTemporaryFile creation
+// on Windows the file could not be written as long as QTemporaryFile owner of the file.
+
+class TemporaryFileWrapper {
+public:
+    static TemporaryFileWrapper create() {
+        QTemporaryFile tempFile(QString("%1/%2_%3")
+                                    .arg(
+                                        QDir::tempPath(),
+                                        qApp->applicationName().isEmpty() ? "qt_temp"
+                                                                          : qApp->applicationName()
+                                    )
+                                    .arg(qApp->applicationPid()));
```

Comment:
According to https://doc.qt.io/qt-6/qtemporaryfile.html#QTemporaryFile the default constructor creates unique filename. Doesn't this work on Windows the same way?

Reply:
This code was to ensure that it behaves equal between qt5 and qt6 and your concerns about running tests in parallel. 
QTemporaryFile between Qt5 and Qt6 seems to differ in default temp file name